app/assets/stylesheets/application.scss

```diff
@@ -1,16 +1,21 @@
 @import 'bootstrap';
```

Comment:
Have you considered changing the import order here? 

Perhaps this order could let us define BS variables before they are evaluated?
```sass
@import 'sul-variables';
@import 'bootstrap-variables';
@import 'bootstrap';
```

Reply:
✅ it works!